unison-core/src/Unison/Referent.hs

```diff
@@ -135,7 +123,3 @@ fold :: (r -> a) -> (r -> Int -> ConstructorType -> a) -> Referent' r -> a
 fold fr fc = \case
   Ref' r -> fr r
   Con' r i ct -> fc r i ct
-
-instance Hashable Referent where
-  tokens (Ref r) = [H.Tag 0] ++ H.tokens r
```

Comment:
Hmm, where'd this `Hashable` instance go? Just want to make sure we aren't accidentally introducing any hashing changes.

Reply:
Oh, nevermind, I see, `Referent'` was just split out into its own module.